### PR TITLE
Add Artifactory Install and Configure Documentation

### PR DESCRIPTION
**Pull request by samuel-rubin**
[skip ci]
Signed-off-by: Samuel Rubin <samuel.rubin@ibm.com>

### CODE REVIEW

**Comment by samuel-rubin** [comment]
@AdamBrousseau for review

File: buildenv/jenkins/ARTIFACTORY.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+sudo apt-get install certbot python-certbot-nginx
+```
+Then I used the certbot automated script to add the certificates and to automatically renew the certificates: 
+```
+sudo certbot –nginx
+```
+Follow the onscreen instructions and everything should work out. 
+
+# Open the Ports
+Log into OpenStack and go to network security groups
```
Comment (AdamBrousseau) [inline buildenv/jenkins/ARTIFACTORY.md]: Should add openstack link
https://openpower-controller.osuosl.org/auth/login/?next=/project/

File: buildenv/jenkins/ARTIFACTORY.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+To start the Artifactory server, run the command:
+```
+/home/jenkins/artifactory/artifactory-oss-6.5.2/bin/artifactoryctl start
+```
+
+On your local browser, go to `http://140-211-168-230-openstack.osuosl.org/` and follow the onscreen instructions.
+The admin password is in the secrets repo and the repository that is used is ‘generic’ with the name being “ci-eclipse-openj9”. 
+
+Next, click on the `Welcome, admin` tab and select `Add User`
+Insert the username `Jenkins` and the email address `j9build@ca.ibm.com`. Insert the password as indicated in the secrets file.
```
Comment (AdamBrousseau) [inline buildenv/jenkins/ARTIFACTORY.md]: Is the email ever needed/used? We may want to use the genie-openj9 email?

File: buildenv/jenkins/ARTIFACTORY.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+# Install Artifactory:
```
Comment (AdamBrousseau) [inline buildenv/jenkins/ARTIFACTORY.md]: Perhaps a short intro paragraph about Artiactory and our usage. Also what machine we have it installed on.

File: buildenv/jenkins/ARTIFACTORY.md (hunk 2 above)
Comment (samuel-rubin) [reply buildenv/jenkins/ARTIFACTORY.md]: I do not believe that the email will be used.

**Pull request merged**